_inc/client/security/backups-scan.jsx

```diff
@@ -96,10 +144,21 @@ export const BackupsScan = moduleSettingsForm(
 			}
 
 			return cardText;
-		};
+		}
 
 		render() {
 			const scanEnabled = get( this.props.vaultPressData, [ 'data', 'features', 'security' ], false );
+			const rewindActive = 'active' === get( this.props.rewindStatus, [ 'state' ], false );
+			const isFetchingData = this.props.isFetchingSiteData || this.props.isFetchingVaultPressData || this.props.isFetchingRewindData;
```

Comment:
I'd like to see these checkings some time since they only add bloat to the app because they can be replaced with the data that we need to use.